app/main/posts/modify/post-editor.directive.js

```diff
@@ -155,18 +154,6 @@ function PostEditorController(
                         })
                         .filter()
                         .value();
-
-                    // adding category-objects to children
-                    _.each(attr.options, function (category) {
```

Comment:
I think this also had the effect of filters the tags to only those specified in attr.options. Have we lost that now? or does it still happen elsewhere?

Reply:
It happens in post-value-edit-directive instead.